packages/react-router-fs-routes/rollup.config.js

```diff
@@ -36,11 +37,13 @@ module.exports = function rollup() {
           babelHelpers: "bundled",
           exclude: /node_modules/,
           extensions: [".ts"],
+          ...remixBabelConfig,
         }),
         typescript({
           tsconfig: path.join(__dirname, "tsconfig.json"),
           exclude: ["__tests__"],
           noEmitOnError: !WATCH,
+          noForceEmit: true,
```

Comment:
Needed to avoid TS incorrectly exporting a CJS module during the rollup esm build